helm/idol-library/templates/_ingress.tpl

```diff
@@ -81,6 +87,21 @@ spec:
 {{- end -}}
 {{- end }}
 
+{{- define "idol-library.ingress.secret" -}}
+{{/*
+  TLS secret containing base64 encoded certificate and private key
+*/}}
+{{- $tls := get . "tls" | required "missing tls" -}}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ $tls.secretName }}
+  namespace: default
```

Comment:
```suggestion
```
Shouldn't need to explicitly state a namespace here - it will automatically go in the namespace the helm chart is deployed in?